docs/features/analysis.md

```diff
@@ -356,7 +356,7 @@ templates together. The controller combines the `metrics` and `args` fields of a
     The controller will error when merging the templates if:
 
     * Multiple metrics in the templates have the same name
-    * Two arguments with the same name both have values
+    * Two arguments with the same name have different default values no matter the argument value in Rollout
```

Comment:
I failed to understand this completely. Is this more clear by any chance?

_"Two arguments with the same name have different default values. It doesn't matter what the argument values are in the Rollout."_

Reply:
The original text is wrong in that if `Two arguments have different default values`, the merging is valid. So the updated text adds the additional condition of `have different default values`

Reply:
Got it. Thanks for explaining :)